homework/questions.txt

```diff
@@ -3,72 +3,83 @@ we went through in class.
 
 1. Navigating the terminal
   a, What is a root directory?
-
+     Root directory is probably the first and top most directory in the hierarchy. The root directory is denoted by (/)slash sign
   b, What is the difference between root and home directory?
-
+     Root directory is reffered to / sign and the topmost directory and the home directory comes under the root directory.
   c, What is the difference between absolute and relative path?
-
+   Absolute path is specifting the location of the file form the root directory and it's the complete path from the start of the root directory. And Relative path is       actually reffered as the presently working directory.
   d, What command do you use to see what directory are you currently in?
-
+   pwd
   e, How would you see the files and folders in a directory?
-
+   ls 
+   ls -a
   f, How do you go back to your home directory?
-
-  g, How do you create a new folder, can you also specify in the command where you want to save it ?
-
+   Using the cd command ( cd ~)
+  g, How do you create a new folder, can you also specify in the command where you want to save it ? 
+    command ( mkdir )
```

Comment:
yes, and you can also specify the location where you would want to create the directory
fx. mkdir  ../some_previous_folder/anotherfolder_inside/my_newfolder